### ISSUE TITLE
Yarn setup scripts throw errors after first use

### ISSUE THREAD

**Issue by denalimarsh** [cosmos/amino-js#8]
The `yarn:setup` script as presented in the README worked during the initial project setup, but the ordering generates errors if the cmd is run again. I'm able to get around the errors by cherry-picking the functionality. Although these issues will likely be avoided if users only run `yarn:setup` once, the scripts may need to be updated to avoid confusion.

Scripts from package.json:
```        
"clean":     "rm -rf ./dist/*",
"dirs":      "mkdir ./dist",
"dist:dev":  "yarn clean && yarn go && NODE_ENV=development yarn build",
"setup":     "yarn dirs && yarn dist:dev",
```

Cmd: `yarn setup`
Issue: `yarn dirs` makes ./dist dir before it's cleaned by `yarn dist:dev`
Result: `mkdir: ./dist: File exists -> error Command failed with exit code 1.`

Cmd: `yarn clean`
Issue: `yarn clean` doesn't remove ./dist dir
Result: Attempting to clean first before cherry-picking `yarn dist:dev` resulted in the same error as above. Was able to resolve by running `rm -rf ./dist` prior to `yarn setup`

**Comment by jordansexton** [comment]
Fixed and published as `0.6.1` / [v0.6.1](https://github.com/cosmos/amino-js/releases/tag/v0.6.1)

**Comment by jordansexton** [comment]
It looks like the original issue may have been from an older package.json

This was already fixed in https://github.com/cosmos/amino-js/commit/d10ccb695acfb2ce64477d3ddd9231c4dbcf9a2a and published as early as [v0.5.1](https://github.com/cosmos/amino-js/releases/tag/v0.5.1)
